src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,31 @@
+package duke.task;
+import java.util.ArrayList;
+
+public class TaskList {
+    /** An arraylist of tasks */
+    private ArrayList<Task> tasks = new ArrayList<>();
+
+
+    /**
+     * Class constructor specifying an arraylist of tasks.
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * Class constructor.
+     */
+    public TaskList() {
```

Comment:
Perhaps you could have initialized the ArrayList in the constructor since that seems more of a standard practice.